### ISSUE TITLE
404 plugin

### ISSUE THREAD

**Issue by WPS12TA** [YOURLS/404-if-not-found#7]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/YOURLS/.github/blob/main/CODE_OF_CONDUCT.md).

### Submission validity

- [X] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions).
- [X] I checked [current issues](https://github.com/YOURLS/YOURLS/issues) and this request isn't a duplicate of an existing issue, opened or closed.

### Self troubleshooting

- [X] I've read the [documentation](https://yourls.org/) and made sure to follow it.
- [X] I've read the [troubleshooting first steps](https://docs.yourls.org/guide/troubleshooting/first-steps.html) and [frequent issues](https://docs.yourls.org/guide/troubleshooting/frequent-issues.html) guides.

### Version

1.8.2

### Description

404 plugin when active stops Preview URL from working normally. 404 cuts in 1st :'-( with a not found message for a valid preview URL.

### Expectation

Both to work normally and appropriately 

### Reproduction steps

Install both and activate both then try to get a URL  preview. You will get a 404.

**Comment by ozh** [comment]
Did you try with the latest version of 404-if-not-found ?

**Comment by WPS12TA** [comment]
Yes. I just did and I could only see the preview URL if the 404 plugin was
deactivated first. When 404 was deactivated preview works fine.

**Comment by UTCGilligan** [comment]
~~404-if-not-found plugin also interferes with seans-qr-code plugin, which generates a QR code at https://sho.rt/abc.qr~~

See issue #1

**Comment by UTCGilligan** [comment]
If you are using Sean's QR code plugin:
- https://github.com/seandrickson/YOURLS-QRCode-Plugin/pull/19/files
